Sloth.Core/Services/CyberSourceBankReconcileService.cs

```diff
@@ -151,13 +156,14 @@ private async Task ProcessReport(Report report, string reportXml, Integration in
 
                         transaction = new Transaction()
                         {
-                            Source                  = integration.Source,
-                            Status                  = TransactionStatuses.Scheduled,
-                            DocumentNumber          = documentNumber,
-                            KfsTrackingNumber       = kfsTrackingNumber,
-                            MerchantTrackingNumber  = deposit.MerchantReferenceNumber,
-                            ProcessorTrackingNumber = deposit.RequestID,
-                            TransactionDate         = deposit.LocalizedRequestDate,
+                            Source                    = integration.Source,
+                            Status                    = TransactionStatuses.Scheduled,
+                            DocumentNumber            = documentNumber,
+                            KfsTrackingNumber         = kfsTrackingNumber,
+                            MerchantTrackingNumber    = deposit.MerchantReferenceNumber,
+                            ProcessorTrackingNumber   = deposit.RequestID,
+                            TransactionDate           = deposit.LocalizedRequestDate,
+                            CybersourceBankReconcileJobRecordId = jobRecord.Id
```

Comment:
FWIW the formatting looks a little off here now, the assignments don't line up.  I think John had a plugin that would do that automatically.  I don't care as much about it, but it does look weird now that all of them are lined up except the last one.

Reply:
I use Code Alignment I think.

Reply:
In this particular example, I had aligned it, but the property didn't contain the word "Reconcile". I later refactored it elsewhere, and boom.